packages/fsweb/proxy/server.prod.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const Proxy = require('./proxy');
+
+const envPath = process.env.ENV_PATH || '../../env';
+const env = require(
+  path.resolve(__dirname, '../../', envPath)
+);
+const port = process.env.PORT || 3000;
+
+if (env && env.dataSource.type === 'commercecloud') {
+  console.log('Env not found in path, ', envPath);
```

Comment:
This shows up every time the server starts and looks like an error message.

Reply:
@Cauldrath done